lib/openhab/core/types/quantity_type.rb

```diff
@@ -274,6 +294,26 @@ def #{ruby_op}(other)
           )
         end
 
+        #
+        # @return [Duration] a {Duration} if the unit is time-based
+        # @return [nil] if the unit is not time-based
+        #
+        def to_duration
+          Duration.of_nanos(to_unit("ns").to_i) if unit.compatible?(Units::SECOND)
+        end
+
+        #
+        # @return [Duration] a {Duration} if the unit is time-based
+        # @raise [TypeError] if the unit is not time-based
+        #
+        # @see CoreExt::Java::TemporalAmount#to_temporal_amount
+        #
+        # @!visibility private
```

Comment:
I don't think this method needs to be hidden

Reply:
it's not hidden

<img width="417" alt="image" src="https://github.com/user-attachments/assets/b29b8b16-d0ca-4ce7-9b95-1572dde990af" />

Reply:
I'll add a description though.

Reply:
I've removed the private visibility, and removed to_duration.